src/components/Newsletter/Newsletter.tsx

```diff
@@ -43,6 +43,7 @@ const Newsletter: FC = () => {
                 <button
                     className={ButtonStyles.button}
                     type="submit"
+                    onClick={onSubmit}
```

Comment:
`happy-dom` has a [bug](https://github.com/capricorn86/happy-dom/issues/527) where a `form` is not submitted when a `button` is clicked.